src/FormInterface.php

```diff
@@ -4,21 +4,241 @@
 
 namespace Yiisoft\Form;
 
-interface FormInterface
+use Yiisoft\Validator\DataSetInterface;
+
+interface FormInterface extends DataSetInterface
 {
-    public function addError(string $attribute, string $error): void;
+    /**
+     * Adds a list of errors.
+     *
+     * @param array $items a list of errors. The array keys must be attribute names.
+     * The array values should be error messages. If an attribute has multiple errors, these errors must be given in
+     * terms of an array.
+     *
+     * You may use the result of {@see errors()} as the value for this parameter.
+     */
     public function addErrors(array $items): void;
-    public function attributeLabels(): array;
+
+    /**
+     * Adds a new error to the specified attribute.
+     *
+     * @param string $attribute attribute name
+     * @param string $error new error message
+     */
+    public function addError(string $attribute, string $error): void;
```

Comment:
What's the use case for adding errors to the form externally if the validation is built-in?

Reply:
It allows to save the validator errors, to later obtain them in the forms.

Reply:
But isn't the validator part of the form model?

Reply:
done.